README_v2.md

```diff
@@ -0,0 +1,176 @@
+
+There is also a gradle task for building custom Solr images from your local checkout.
+These local images are built identically to the official image except for retrieving the Solr artifacts locally instead of from the official release.
+This can be useful for testing out local changes as well as creating custom images for yourself or your organization.
+The task will output the image name to use at the end of the build.
+
+`./gradlew docker`
+
+For more info on building an image, run:
+
+`./gradlew helpDocker`
+
+### Running Solr on Kubernetes
+
+Solr has official support for running on Kubernetes, in the official Docker image.
+Please refer to the [Solr Operator](https://solr.apache.org/operator) home for details, tutorials and instructions.
+
+## Building Solr from Source
+FIXME --> is this where we get JDK from??   What about OpenJDK?
+Download the Java 11 JDK (Java Development Kit) or later from https://jdk.java.net/
+You will need the JDK installed, and the $JAVA_HOME/bin (Windows: %JAVA_HOME%\bin)
+folder included on your command path. To test this, issue a "java -version" command
+from your shell (command prompt) and verify that the Java version is 11 or later.
+
+Download the Apache Solr distribution, from http://solr.apache.org/.  FIXME link??
+Unzip the distribution to a folder of your choice, e.g. C:\solr or ~/solr
+Alternately, you can obtain a copy of the latest Apache Solr source code
+directly from the Git repository:
+
+<https://solr.apache.org/community.html#version-control>
+
+As of 9.0, Solr uses [Gradle](https://gradle.org/) as the build
+system.  Navigate to the root of your source tree folder and issue the `./gradlew tasks`
+command to see the available options for building, testing, and packaging Solr.
+
+`./gradlew assemble` will create a Solr executable.
+cd to `./solr/packaging/build/solr-9.0.0-SNAPSHOT` and run the `bin/solr` script
+to start Solr.
+
+NOTE: `gradlew` is the "Gradle Wrapper" and will automatically download and
+start using the correct version of Gradle for Solr.
+
+NOTE: `./gradlew help` will print a list of high-level tasks. There are also a
+number of plain-text files in <source folder root>/help.
+
+The first time you run Gradle, it will create a file "gradle.properties" that
+contains machine-specific settings. Normally you can use this file as-is, but it
+can be modified if necessary.
+
+Note as well that the gradle build does not create or copy binaries throughout the
+source repository so you need to switch to the packaging output folder `./solr/packaging/build`;
+the rest of the instructions below remain identical. The packaging directory
+is rewritten on each build.
+
+If you want to build the documentation, type `./gradlew -p solr documentation`.
+
+`./gradlew check` will assemble Solr and run all validation tasks unit tests.
+
+Lastly, there is developer oriented documentation in `./dev-docs/README.adoc` that
+you may find useful in working with Solr.
+
+
+### Gradle build and IDE support
```

Comment:
General about building from source and gradle - perhaps we should make some really good and thorough guides about that topic in `dev-docs`, and just have a few really basic examples here in the main readme, like `./gradlew dev` and `cd solr/packaging/dev/ && bin/solr -e techproducts`...

But i see that the main focus here is merging the two READMEs, so feel free to postpone dev-docs work to later.

Reply:
That can be done in followup issues. I think we plan for a separate dev-guide, and not mash everything into ref-guide.